From 23b4d5fe59c1ea3552af30f184a77768a2d65723 Mon Sep 17 00:00:00 2001
From: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Date: Wed, 21 Apr 2021 13:43:52 +0200
Subject: [PATCH 0242/1141] dt-bindings: remoteproc: Add compatibility for
 STM32MP2 support

Add the compatibility with the STM32MP2 Socs.

Signed-off-by: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Change-Id: Iec76f6abbeea79116d00f5a92ec5780ed820f322
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/244376
Reviewed-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Tested-by: Alexandre TORGUE <alexandre.torgue@foss.st.com>
---
 .../devicetree/bindings/remoteproc/st,stm32-rproc.yaml        | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

--- a/Documentation/devicetree/bindings/remoteproc/st,stm32-rproc.yaml
+++ b/Documentation/devicetree/bindings/remoteproc/st,stm32-rproc.yaml
@@ -16,7 +16,9 @@ maintainers:
 
 properties:
   compatible:
-    const: st,stm32mp1-m4
+    enum:
+      - st,stm32mp1-m4
+      - st,stm32mp2-m33
 
   reg:
     description:
